frontend/src/metabase/setup/components/InactiveStep/InactiveStep.styled.tsx

```diff
@@ -14,12 +14,11 @@ export const StepRoot = styled.section<Props>`
   padding: 1rem 2rem;
   margin-bottom: 1.75rem;
   background-color: ${props =>
-    props.isCompleted ? color("white") : "var(--mb-color-bg-medium)"};
+    color(props.isCompleted ? "bg-white" : "bg-medium")};
```

Comment:
I'd leave `--mb-color-bg-medium` as is as it should be preferable over any color functions calls